packages/db/index.d.ts

```diff
@@ -38,7 +39,10 @@ export class DBClient {
   getMetricsValue (key: string): Promise<{ total: number }>
   upsertUser (user: UpsertUserInput): Promise<UpsertUserOutput>
   getUser (issuer: string): Promise<UserOutput>
-  getUsedStorage (userId: number): Promise<number>
+  getUsedStorage (userId: number): Promise<{ uploaded: number, pinned: number, total: number }>
```

Comment:
Shall we have all these input/output types to be defined in `db-client-types.d.ts`?
That would avoid having duplication here and in index.js? To have the types a bit easier to mantain.

Reply:
Yep that's a good shout. Have also renamed a bit for clarity/consistency.

Reply:
thanks